packages/libs/components/src/map/MapVEuMap.tsx

```diff
@@ -343,13 +362,19 @@ function MapVEuMapEvents(props: MapVEuMapEventsProps) {
         zoomLevel: mapEvents.getZoom(),
       };
       onBoundsChanged(boundsViewport);
+
+      // close left-side panel when drag/panning event happens
```

Comment:
Same comment about this comment (and the ones above and below). Best to remove.

Reply:
all removed!